CRM/Upgrade/Incremental/sql/5.61.alpha1.mysql.tpl

```diff
@@ -7,4 +7,13 @@ UPDATE civicrm_payment_processor
 UPDATE civicrm_payment_processor
   SET {localize field="title"}title = name{/localize};
 
+UPDATE civicrm_contribution_page
+SET {localize field="frontend_title,title"}frontend_title = title WHERE frontend_title IS NULL OR frontend_title = ''{/localize};
+
+UPDATE civicrm_group
+SET {localize field="frontend_title,title"}frontend_title = title WHERE frontend_title IS NULL OR frontend_title = ''{/localize};
+
+UPDATE civicrm_uf_group
+SET {localize field="frontend_title,title"}frontend_title = title WHERE frontend_title IS NULL OR frontend_title = ''{/localize};
+
```

Comment:
```suggestion
{if $multilingual}
  {foreach from=$locales item=locale}
   UPDATE civicrm_contribution_page
   SET frontend_title_{$locale} = title_{$locale} WHERE frontend_title_{$locale} IS NULL OR frontend_title_{$locale} =  '';

  UPDATE civicrm_group
  SET frontend_title_{$locale} = title_{$locale} WHERE frontend_title_{$locale} IS NULL OR frontend_title_{$locale} = '';

  UPDATE civicrm_uf_group
  SET frontend_title_{$locale} = title_{$locale} WHERE frontend_title_{$locale} IS NULL OR frontend_title_{$locale} = '';
  {/foreach}
{else}
UPDATE civicrm_contribution_page
   SET frontend_title = title WHERE frontend_title IS NULL OR frontend_title = '';
UPDATE civicrm_group
  SET frontend_title = title WHERE frontend_title IS NULL OR frontend_title = '';

UPDATE civicrm_uf_group
  SET frontend_title = title WHERE frontend_title IS NULL OR frontend_title = '';
{/if}
```

This is as per https://docs.civicrm.org/dev/en/latest/translation/database/#sql-upgrades